openreview/venue_request/process/postSubmissionProcess.py

```diff
@@ -1,7 +1,7 @@
 def process(client, note, invitation):
     GROUP_PREFIX = ''
     SUPPORT_GROUP = GROUP_PREFIX + '/Support'
-    conference = openreview.helpers.get_conference(client, note.forum, SUPPORT_GROUP, setup=False)
+    conference = openreview.helpers.get_conference(client, note.forum, SUPPORT_GROUP, setup=True)
```

Comment:
@melisabok is there a reason why setup is `False` here? I noticed that if, for example, the submissions readers are changed to `everyone` using the post submission button, the content of the venue group is not updated (for example `public_submissions`)

Or should I just update `public_submissions` in the group content instead?

Reply:
I didn't want to setup the whole submission stage again but I'm realizing that is not happening in the `setup` function, the call to create submission stage is being done in the "deploy" and "revision" process functions, so you can change it to true.